spec/index.md

```diff
@@ -641,7 +645,11 @@ The HTTP response to this request consists of a CBOR (see [CBOR](#cbor)) map wit
 
     If this `certificate` includes subnet delegations (possibly nested), then the `effective_canister_id` must be included in each delegation's canister id range (see [Delegation](#certification-delegation)).
 
-The returned certificate reveals all values whose path is a suffix of a requested path. It also always reveals `/time`, even if not explicitly requested.
+The returned certificate reveals all values whose path has a requested path as a prefix except for
+
+-   paths with prefix `/subnet/<subnet_id>/nodes`. Only contained in the returned certificate if `<effective_canister_id>` belongs to the canister ranges of the subnet `<subnet_id>`, i.e., if `<effective_canister_id>` belongs to the value at the path `/subnet/<subnet_id>/canister_ranges` in the state tree.
```

Comment:
There are restrictions on paths with prefix `/subnet/<subnet_id>/nodes` but  `/subnet` can always be requested.

This seems to make the restrictions void as `/subnet` contains `/subnet/<subnet_id>/nodes`.

Shall we apply the restriction while reading `/subnet`?

Reply:
Never mind. I misunderstand this sentence. It does mean the results of `/subnet` follow this rule.